src/azure-cli/azure/cli/command_modules/cloud/tests/latest/test_cloud.py

```diff
@@ -108,7 +104,7 @@ def test_cloud_scenario(self):
 
         # TODO: Test all arguments of `az cloud update`
 
-        self.cmd('cloud set --name {name} --profile 2020-09-01-hybrid')
+        self.cmd('cloud set --name {name} --profile latest')
```

Comment:
I'm wondering if we should remove `--profile` argument for `az cloud set` and `az cloud list-profile` command

Reply:
In this PR only built-in azure-stack profiles are removed. It still keep the ability for profile switch and registration.